### ISSUE TITLE
Receiving the following error from HA after 2024.6 upgrade this morning.

### ISSUE THREAD

**Issue by georgelza** [sabeechen/hassio-google-drive-backup#1052]
<img width="858" alt="Screenshot 2024-06-06 at 10 41 17" src="https://github.com/sabeechen/hassio-google-drive-backup/assets/28488383/117052a5-a6cb-4121-8220-d4ba77efc090">

please advise.

G

**Comment by kylevidrine** [comment]
I received the same message.

**Comment by zalmyr** [comment]
Same here

**Comment by georgelza** [comment]
fixed it by re-adding the repo and reinstalling the app.
G

**Comment by LanceMcCarthy** [comment]
It seems the addon was broken for a while, not until 2024.6 was I made aware 😱  I did the same as @georgelza and got back to a good state.

> [!NOTE]
> Don't be scared to delete it locally. YEs, you will go and generate a new token, but it was a 30 second re-setup experience to choose the settings again.

**Comment by DarcyDetlor** [comment]
I stopped, uninstalled, re-started, re-installed, and re-configured Google Drive Backup and I am still getting the same error.

<img width="741" alt="image" src="https://github.com/sabeechen/hassio-google-drive-backup/assets/87451444/8cc089d5-7df4-4e94-bf18-88eb86332546">

How are we supposed to fix this problem?

**Comment by georgelza** [comment]
the problem was not the program, it was the link to the repo that got broken.
you need to read the repo url first under custom repo's.
then do the reinstall. that's what solved it for me. had another add-on paradox alarm integration with exact same, solved same way.

**Comment by DarcyDetlor** [comment]
> the problem was not the program, it was the link to the repo that got broken. you need to read the repo url first under custom repo's. then do the reinstall. that's what solved it for me. had another add-on paradox alarm integration with exact same, solved same way.

Thank you for the reply, but I do not understand what you are saying.  "you need to read the repo URL first under custom repo's".  Where is the repo URL?

I have it installed here: 

<img width="630" alt="image" src="https://github.com/sabeechen/hassio-google-drive-backup/assets/87451444/88b53ce5-2331-41ae-867c-8229d06a184a">

After I find the repo URL, you suggest that I can then reinstall?  How is this different from uninstalling and reinstalling according the the author's instructions?

**Comment by georgelza** [comment]
hmm let me try again...

I needed to re-add the repo... for that go to system/add-ons and then top right click on 3 ...
then add custom repo and reenter the url from the install site.

you will find if you look at your url's at the moment the repo needed for the back util is gone.

once you've done this you can re-install.

G

**Comment by igorsantos07** [comment]
I didn't need to reinstall it nor reconfigure anything - just adding the correct add-on URL made HA find the new version (0.112.1, I was stuck at 0.105.2) and it updated just fine.

**Comment by cherryocola** [comment]
> I didn't need to reinstall it nor reconfigure anything - just adding the correct add-on URL made HA find the new version (0.112.1, I was stuck at 0.105.2) and it updated just fine.

I re-added the URL but I am still stuck on 0.105.2, is there anyway to force a manual update?

**Comment by georgelza** [comment]
if i recall i deleted the google app... and reinstalled after i readied the repo url as a custom repo to have available.

G

**Comment by jmcbri** [comment]
Thanks @igorsantos07 !:
> I didn't need to reinstall it nor reconfigure anything - just adding the correct add-on URL made HA find the new version (0.112.1, I was stuck at 0.105.2) and it updated just fine.

>> But where and what URL do you add? Per the Github page, I guess it's https://github.com/sabeechen/hassio-google-drive-backup but, to where?  I can't find a configuration setting, even in yaml, that would let me do that.

**Comment by LanceMcCarthy** [comment]
@jmcbri 

![Screenshot_20240619_173244_Home Assistant.jpg](https://github.com/sabeechen/hassio-google-drive-backup/assets/3520532/68e39c1b-a17e-4eb9-bf8e-aeae1d9325e9)

**Comment by jmcbri** [comment]
> @jmcbri
> 
> ![Screenshot_20240619_173244_Home Assistant.jpg](https://private-user-images.githubusercontent.com/3520532/341208928-68e39c1b-a17e-4eb9-bf8e-aeae1d9325e9.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MzMxMjUsIm5iZiI6MTcxODgzMjgyNSwicGF0aCI6Ii8zNTIwNTMyLzM0MTIwODkyOC02OGUzOWMxYi1hMTdlLTRlYjktYmY4ZS1hZWFlMWQ5MzI1ZTkuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MTlUMjEzMzQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjEwN2RjYWQ3Yzg1Yjk4NDA3MjdkNTUyZThkN2NkZjg4Njg1ZTMwZDA2NWRmMWE3YTlkN2RjMzY0Y2RkMWFmMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.nU0txhPnr6u6N1rZ_NpzT2gx3WTvSICujDYdb0dWlKk)

I wish people were more responsive!  I had to minute more than 10 seconds for that! (I'm kidding, that was fantastic.)  I did that.   Thanks.

**Comment by georgelza** [comment]
...if i recall i did say click add on stop bottom, then top right click 3 dots, then go repositories and then add your url and click add... 

but happy all fixed.

**Comment by foltymat** [comment]
I can hopefully put an end to this thread as I was dealing with the issue myself.
Should anyone face the problem, do the following:

1. Open Settings -> Add-ons
2. Three dots in the upper right corner -> Repositories
3. Add the URL mentioned in this very github - https://github.com/sabeechen/hassio-google-drive-backup
4. Go to the Google Backup addon
5. Uninstall
6. Install
7. Authenticate with Google again
8. The addon finds the original folder and just continues wherever you left off
9. Hit ignore on the pesky Repair message (seems to not get cleared automatically)
10. Restart Home Assistant
11. Pour yourself a cup of tea and pat yourself on the back because this took you 90 seconds, including the reboot.
